docs/source/dashboards/index.rst

```diff
@@ -0,0 +1,17 @@
+======================================
+Grafana Dashboards Configuration Guide
```

Comment:
This is not following the tree in https://github.com/scylladb/scylla-monitoring/issues/1346

Reply:
I didn't see that issue when I started this PR. I'll make the changes accordingly.

Reply:
once we're happy with the new organization, I'll make the redirections. We currently don't have a redirections file so I'll have to add one.